back/tests/openbsrv/common_test.go

```diff
@@ -0,0 +1,58 @@
+package artifact_tests
```

Comment:
Please rename package declarations to `main_test`.

Reply:
Done.